src/wwwroot/partials/settings/my-plan.html

```diff
@@ -7,21 +7,21 @@
   </div>
   <div class="section--wrapper my-plan--info-container">
     <div class="item flex flex--wrap first">
-      <div class="flex one flex--wrap">
+      <div class="flex one flex--wrap current-plan-price">
```

Comment:
I understand that a class only for tests is a smell, but, in my opinion, a data attribute only for tests is not so different.

In my opinion, class attribute _is not for styles_ is for express semantics. Then, depending on the styles, that semantic could be represented with a special style or not. 

So, if the desired class express the semantics of the element, I mean, if this class could help a machine to understand the semantics of an element I think that it is a valid class. A machine could be a browser selecting the right style or Protractor trying to get an element with a particular meaning.

Of course, it could be better if we are consistent with the naming and how we represent those semantics using classes.